src/en/mangago/src/eu/kanade/tachiyomi/extension/en/mangago/Mangago.kt

```diff
@@ -151,11 +162,21 @@ class Mangago : ParsedHttpSource() {
 
     override fun searchMangaNextPageSelector() = genreListingNextPageSelector
 
+    private val titleRegex = Regex("""\(yaoi\)|\{Official\}|«Official»|〘Official〙|\(Official\)|\s\[Official]|\s「Official」|『Official』|\s?/Official\b""", RegexOption.IGNORE_CASE)
+    private fun titleVersion(title: String) = title.replace(titleRegex, "").substringBeforeLast("(").trim()
+
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
-        title = document.selectFirst(".w-title h1")!!.text()
+        if (isRemoveTitleVersion()) {
+            title = titleVersion(document.select(".w-title h1").text())
+        }
```

Comment:
```suggestion
        title = document.selectFirst(".w-title h1")!!.text()
        if (isRemoveTitleVersion()) {
            title = titleVersion(title)
        }
```
set the title regardless of setting